### PR TITLE
Docker issue

### PR DESCRIPTION
**Pull request by Vghxv**
### Misc Changes
- now backend waits for mysql loaded before connecting to database

### CODE REVIEW

**Comment by Vghxv** [comment]
urgent fix

File: wait.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+until nc -z -v -w30 "nftbackend-mysql-1.nftbackend_default" 3306
+do
+  echo "Waiting a second until the database is receiving connections..."
+  # wait for a second before checking again
+  sleep 1
+done
```
Comment (PEKKA1117) [inline wait.sh:6]: https://codefather.tech/blog/bash-unexpected-end-of-file/

some Windows new line characters breaks the script(CRLF-LF shit).


```bash
# vim wait.sh -b
until nc -z -v -w30 "nftbackend-mysql-1.nftbackend_default" 3306^M 
do^M 
  echo "Waiting a second until the database is receiving connections..."^M 
  # wait for a second before checking again^M 
  sleep 1^M 
done^M
```

File: wait.sh [hunk 2]
```diff
@@ -0,0 +1,6 @@
+until nc -z -v -w30 "nftbackend-mysql-1.nftbackend_default" 3306
```
Comment (PEKKA1117) [inline wait.sh]: "Additionally, the hostname `nftbackend-mysql-1.nftbackend_default` doesn't appear to be the correct database hostname. The script functions properly after replacing the hostname with  'mysql'."

**Review by Dragonfly911117 — CHANGES_REQUESTED** [review]

File: wait.sh (hunk 2 above)
Comment (Vghxv) [reply wait.sh]: Guess didn't clean my old network then

**Review by Dragonfly911117 — APPROVED** [review]

**Pull request merged**